lib/src/header_parser/sub_parsers/enumdecl_parser.dart

```diff
@@ -24,12 +24,13 @@ class _ParsedEnum {
 
 final _stack = Stack<_ParsedEnum>();
 
-/// Parses a function declaration.
```

Comment:
Copy paste comments. How did you sneak this by me in a previous PR. 😄 

(I of course [~~never~~](https://dart-review.googlesource.com/c/sdk/+/200188/13..15/benchmarks/MapLookup/dart/MapLookup.dart#b48) do that.)

Reply:
🙈